modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,300 @@
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ultra-disk-sc <1>
+parameters:
+  cachingMode: None
+  diskIopsReadWrite: "2000" <2>
+  diskMbpsReadWrite: "320" <3>
+  kind: managed
+  skuname: UltraSSD_LRS
+provisioner: disk.csi.azure.com <4>
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <5>
+----
+<1> Specify the name of the storage class. This procedure uses `ultra-disk-sc` for this value.
+<2> Specify the number of IOPS for the storage class.
+<3> Specify the throughput in MBps for the storage class.
+<4> For Azure Kubernetes Service (AKS) version 1.21 or later, use `disk.csi.azure.com`. For earlier versions of AKS, use `kubernetes.io/azure-disk`.
+<5> Optional: Specify this parameter to wait for the creation of the pod that will use the disk.
+
+. Create a persistent volume claim (PVC) to reference the `ultra-disk-sc` storage class that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ultra-disk <1>
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: ultra-disk-sc <2>
+  resources:
+    requests:
+      storage: 4Gi <3>
+----
+<1> Specify the name of the PVC. This procedure uses `ultra-disk` for this value.
+<2> This PVC references the `ultra-disk-sc` storage class.
+<3> Specify the size for the storage class. The minimum is 4Gi.
+
+. Create a pod that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-ultra
+spec:
+  nodeSelector:
+    disk: ultrassd <1>
+  containers:
+  - name: nginx-ultra
+    image: alpine:latest
+    command:
+      - "sleep"
+      - "infinity"
+    volumeMounts:
+    - mountPath: "/mnt/azure"
+      name: volume
+  volumes:
+    - name: volume
+      persistentVolumeClaim:
+        claimName: ultra-disk <2>
+----
+<1> Specify the label of the machine set that enables the use of ultra disks. This procedure uses `disk.ultrassd` for this value.
+<2> This pod references the `ultra-disk` PVC.
+endif::pvc[]
+
+.Verification
+
+. Validate that the machines are created by running the following command:
++
+[source,terminal]
+----
+$ oc get machines
+----
++
+The machines should be in the `Running` state.
+
+. For a machine that is running and has a node attached, validate the partition by running the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node-name> -- chroot /host lsblk
+----
++
+In this command, `oc debug node/<node-name>` starts a debugging shell on the node `<node-name>` and passes a command with `--`. The passed command `chroot /host` provides access to the underlying host OS binaries, and `lsblk` shows the block devices that are attached to the host OS machine.
+
+.Next steps
+
+* To use an ultra disk from within a pod, create workload that uses the mount point. Create a YAML file similar to the following:
++
```

Comment:
Maybe `Create a YAML file similar to the following example:` just to clarify thats in an example yaml. Unless its not an example, so its up to you 😃